### ISSUE TITLE
please remove Storytlr domains

### ISSUE THREAD

**Issue by NicoleSharp2000** [eschnou/storytlr#105]
Can someone please remove the hyperlinks to <storytlr.org> and <storytlr.com> from the open-source application?  The Storytlr domains have been expired or hacked, and those sites now display anti-Semitic hate speech and pornographic content.  <eschenauer.be> also displays the same content as a mirror of <storytlr.org>.

**Comment by eschnou** [comment]
Thanks for the notifiction. Seems indeed of a DNS hack. No clue how long this has been there. I'll try to resolve the matter asap. Thanks again for letting me know.

**Comment by eschnou** [comment]
Situation has been resolved. I had decommissioned the server hosting these pages a few months ago and never properly cleaned up the DNS records. Seems te IP eventually got allocated to another host and the domain started to point to this content. It looks like the new content started to appear around February 11th. It doesn't look like a targeted hack and I'm not even sure the new owner was aware of that.

Anyway, domains are backup with updated links and content. Thanks again @nicolesharp100 for taking the time to report this issue.
